src/components/Text/Text.stories.tsx

```diff
@@ -1,10 +1,13 @@
 import type { StoryObj } from "@storybook/react";
+import clsx from "clsx";
 import React from "react";
 
-import Text from "./Text";
+import { Text } from "./Text";
+import type { Variant } from "./Text";
+import styles from "./Text.stories.module.css";
 
 export default {
-  title: "Text",
+  title: "Atoms/Text/Text",
```

Comment:
```suggestion
  title: "Text",
```
?

Reply:
Yea idk how that happened, will change back